README.md

```diff
@@ -0,0 +1,127 @@
+# Panacea Smart Contract
```

Comment:
I write to document how to create and exchange NFTs (receive rewards) from the initial installation.